charts/taraxa-node/templates/taraxa-node.yaml

```diff
@@ -35,8 +35,16 @@ spec:
     spec:
       initContainers:
         - name: config-adapter
+          {{- if and .Values.node.image.repository .Values.node.image.tag }}
+          image: "{{ .Values.node.image.repository }}:{{ .Values.node.image.tag }}"
+          {{- else }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          {{- end }}
+          {{- if .Values.node.image.imagePullPolicy }}
```

Comment:
Should be `.Values.node.image.pullPolicy`

Reply:
fixed